.travis.yml

```diff
@@ -3,13 +3,13 @@ dist: xenial
 language: python
 
 python:
+  - "3.8"
+  - "3.7"
   - "3.6"
-  - "3.5"
 
 env:
-  - DJANGO="Django==1.11.*"
-  - DJANGO="Django==2.1.*"
-  - DJANGO="Django==2.2.*"
+  - DJANGO="Django~=2.2.8"
```

Comment:
Accoring to Django 2.2's release notes, it supports Python 3.8 as of 2.2.8.
https://docs.djangoproject.com/en/2.2/releases/2.2/#python-compatibility